addOns/graphql/src/main/java/org/zaproxy/addon/graphql/GraphQlFingerprinter.java

```diff
@@ -95,6 +98,9 @@ public void fingerprint() {
             try {
                 if (fingerprinter.getValue().getAsBoolean()) {
                     raiseFingerprintingAlert(fingerprinter.getKey());
+
+                    appConsumer.accept(
```

Comment:
Would be better if exceptions were handled specifically for the consumer (just in case), rather than handle it as a fingerprint error which would not break.